monkey/infection_monkey/monkey.py

```diff
@@ -183,6 +184,7 @@ def _get_local_network_interfaces():
     @staticmethod
     def _build_puppet() -> IPuppet:
         puppet = Puppet()
+        puppet.load_plugin("http", HTTPFingerprinter(), PluginType.FINGERPRINTER)
         puppet.load_plugin("ransomware", RansomwarePayload(), PluginType.PAYLOAD)
 
         return puppet
```

Comment:
Question unrelated to this PR: what is this function supposed to look like finally?

Reply:
At some point it will be a line like this for every plugin, at which point we'll have to decide if we want to use a different construct. That being said, the plan is to turn these plugins into actual plugins, load them into the Island, and have the Island distribute them upon request. The master will then be responsible for requesting plugins from the master and loading them into the agent. Given that change is in the works, we shouldn't spend too much time on this function. As long as it doesn't commit a mortal sin, we can leave it alone.